crypto_libraries/inc/nx_crypto_ecdh.h

```diff
@@ -65,7 +65,7 @@ extern   "C" {
 
 
 /* Max Elliptic-curve Diffie-Hellman key size. Buffer size for calculations is 4X the key size */
-#define NX_CRYPTO_ECDH_MAX_KEY_SIZE     (68)
+#define NX_CRYPTO_ECDH_MAX_KEY_SIZE     (80)  /* Buffer size is increased to accomodate OEM keys */
```

Comment:
I think its better to change this to
```suggestion
#ifndef NX_CRYPTO_ECDH_MAX_KEY_SIZE
#define NX_CRYPTO_ECDH_MAX_KEY_SIZE     (80)  /* Buffer size is increased to accomodate OEM keys */
#endif
```